### ISSUE TITLE
Babel should not be a devDependency

### ISSUE THREAD

**Issue by cadecairos** [mozilla/learning.mozilla.org#2098]
Babel was added as a devDependency to the webpack build in [this commit](https://github.com/mozilla/learning.mozilla.org/commit/ee484455a8a8fed267af00c3f425cf371a742716)

Unfortunately, Heroku's NodeJS buildpack runs `npm install --production` which means that these are build-time depenencies, which means we need to move them into the plain-old dependencies list.

P1 Because production pushes are currently blocked

**Comment by Pomax** [comment]
good catch. I thought I had added it as a regular dependency for exactly that reason but clearly I hadn't
